changes/en-us/2.0.0.md

```diff
@@ -83,7 +83,7 @@ The version is updated as follows:
 - [[#5971](https://github.com/seata/seata/pull/5971)] fix some configurations that are not deprecated show "Deprecated"
 - [[#5977](https://github.com/seata/seata/pull/5977)] fix that rpcserver is not closed when raftServer is closed
 - [[#5954](https://github.com/seata/seata/pull/5954)] fix the issue of saved branch session status does not match the actual branch session status
-
+- [[#5887](https://github.com/seata/seata/pull/5887)] fix global transaction hook repeat execute
```

Comment:
id : 5997 for 2.0.0 ?

Reply:
This bug also needs to be resolved in 2.x